docs/cfn-service-role.yaml

```diff
@@ -11,31 +11,51 @@ Resources:
               Service: cloudformation.amazonaws.com
             Action: sts:AssumeRole
       Policies:
-        - PolicyName: root
+        - PolicyName: QuiltServiceRolePolicy
           PolicyDocument:
             Version: '2012-10-17'
             Statement:
               - Effect: Allow
                 Action:
                   - apigateway:DELETE
                   - apigateway:GET
-                  - apigateway:GetResources
                   - apigateway:PATCH
                   - apigateway:POST
                   - apigateway:PUT
+                  - apigateway:SetWebACL
                   - apigateway:UpdateRestApiPolicy
+                  - appsync:SetWebACL
+                  - apprunner:AssociateWebAcl
+                  - apprunner:DescribeWebAclForService
+                  - apprunner:DisassociateWebAcl
```

Comment:
where does this come from? 🤔

Reply:
That is apparently how WAF operations are dispatched. Specifically it was reported by IAM > Generate Policy.

Reply:
won't it work without this?